Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/AccelerometerSamplePage.xaml.cs

```diff
@@ -173,12 +175,14 @@ private void Accelerometer_Shaken(Accelerometer sender, AccelerometerShakenEvent
 			++ShakeCount;
 		}
 
-		private void Accelerometer_ReadingChanged(Accelerometer sender, AccelerometerReadingChangedEventArgs args)
+		private async void Accelerometer_ReadingChanged(Accelerometer sender, AccelerometerReadingChangedEventArgs args)
 		{
-			LastReadTimestamp = args.Reading.Timestamp;
-			AccelerationX = args.Reading.AccelerationX;
-			AccelerationY = args.Reading.AccelerationY;
-			AccelerationZ = args.Reading.AccelerationZ;
+			await CoreApplication.MainView.CoreWindow.Dispatcher.RunAsync(CoreDispatcherPriority.Normal, () => {
```

Comment:
```suggestion
			_ = CoreApplication.MainView.CoreWindow.Dispatcher.RunAsync(CoreDispatcherPriority.Normal, () => {
```

It's probably best to avoid using `async void`, though even if in particular place it's likely never going to fail, I'd prefer not to promote the pattern :)